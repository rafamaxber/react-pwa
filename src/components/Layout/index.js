import React from 'react';
import styled from 'styled-components';
import Header from '../Header/';

const Container = styled.div`
`;

const Main = styled.main`
  max-width: 800px;
  margin: 0 auto;
`;

export default ({ children }) => (
  <Container>
    <Header></Header>
    <Main>
      { children }
    </Main>
  </Container>
);