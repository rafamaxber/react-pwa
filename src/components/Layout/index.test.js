import React from 'react';
import renderer from 'react-test-renderer';
import Layout from './index';

jest.mock('react-router-dom');

describe('Layout', () => {
  it('Should match with snapshot', () => {
    const component = renderer.create(
      <Layout></Layout>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});