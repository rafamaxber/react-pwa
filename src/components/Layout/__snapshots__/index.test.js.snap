// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Should match with snapshot 1`] = `
<div
  className="sc-bxivhb eBNJKg"
>
  <header
    className="sc-bdVaJa lbQKwt"
  />
  <main
    className="sc-ifAKCX cTCzMl"
  />
</div>
`;
